render/plot.go

```diff
@@ -237,16 +241,29 @@ func (p Plot) Paint(dc *gg.Context, bounds image.Rectangle, frameIdx int) {
 		}
 	}
 
-	// the line itself
-	for i := 0; i < pl.Length(); i++ {
-		x, y := pl.Point(i)
-		if y > p.invThreshold {
-			dc.SetColor(colInv)
-		} else {
-			dc.SetColor(col)
+	if p.ChartType == "scatter" {
+		points := p.translatePoints()
+		for i := 0; i < len(points); i++ {
+			point := points[i]
+			if point.Y > p.invThreshold {
+				dc.SetColor(colInv)
+			} else {
+				dc.SetColor(col)
+			}
+			dc.SetPixel(int(point.X), int(point.Y))
+		}
+	} else {
```

Comment:
I could pass `foo` here and it wouldn't be an error, but that seemingly is how we handle these enum cases in other widgets. @matslina we should consider updating cases in the future so that an incorrect value produces an error.

Reply:
Yea I wanted to add better error handling in this but wanted to keep it consistent with how you all were handling similar logic since the codebase was new to me! Happy to follow up and correct that here!